lib/driver.js

```diff
@@ -155,6 +158,9 @@ class AndroidDriver extends BaseDriver {
     if (key === "ignoreUnimportantViews") {
       await this.setCompressedLayoutHierarchy(value);
     }
+    if (key === "configurator") {
+      await this.setConfiguratorConfig(value);
```

Comment:
what would configurator 'values' look like here? wouldn't they actually be a combination of key and value? are they supposed to be JSON objects?